cost-analyzer/templates/integrations-postgres-queries-configmap.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if .Values.global.integrations.postgres.enabled }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: kubecost-integrations-postgres-queries
+  namespace: {{ .Release.Namespace }}
+  labels:
+    {{ include "cost-analyzer.commonLabels" . | nindent 4 }}
+data:
+    kubecost-queries.json: |-
+      {
+        {{- if .Values.global.integrations.postgres.allocations.enabled }}
+        "allocations": {{- .Values.global.integrations.postgres.allocations.queryConfigs | toJson | nindent 10 }}
+        {{- else }}
+        "allocations": []
```

Comment:
Would we really have to include an empty array if allocations isn't enabled? Or can we just omit entirely?

Reply:
Let me double check how the backend handles cases where the "allocations" key does not exist in `/var/configs/integrations/postgres-queries/kubecost-queries.json`